### ISSUE TITLE
[WebServerBundle] doesn't read configuration when in production.

### ISSUE THREAD

**Issue by ndyakov** [symfony/symfony#26126]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.4.4

We had very strange problem with WebServerBundle. Until we copied the `webserver.xml` configuration in `app/config` the `docroot` and `environment` were not injected in the commands.

**Comment by chalasr** [comment]
Did you clear your prod cache before running the command?

**Comment by ndyakov** [comment]
@chalasr I don't recall if I did. I will try to reproduce the bug later.

**Comment by Simperfit** [comment]
@ndyakov did you reproduce the bug ?

**Comment by ndyakov** [comment]
@simensen , @chalasr
Sorry for the delay. 
The **WebServerBundle seems to work** as expected on my local machine after `cache:clear`. I guess the flow for this to happen was something like:

1. WebServerBundle was only enabled in dev, so I moved it to be enabled in any environment.
2. Try to run it without clearing the cache for prod.

tl;dr
**This is not a bug, works as expected after `cache:clear`**
